zc_common/remote_resource/filters.py

```diff
@@ -27,6 +29,14 @@ def filter_queryset(self, request, queryset, view):
                 if hasattr(queryset.model, field_name)\
                         and isinstance(getattr(queryset.model, field_name).field, ManyToManyField):
                     value = value.split(',')
+
+                # Allow 'true' or 'false' as values for boolean fields
+                try:
+                    if isinstance(queryset.model._meta.get_field(field_name), BooleanField):
+                        value = bool(strtobool(value))
+                except FieldDoesNotExist:
+                    pass
```

Comment:
@birkholz interestingly, a `ManyToManyField` can be retrieved both ways (either through the try/catch or the `getattr`) but a `BooleanField` can only be retrieved through try/catch. This is because `ManyToManyField` is a related field. That is a django's mess because I think the API should be consistent as far as fields are concerned.